ui/lib/pki/addon/components/page/pki-configuration-edit.ts

```diff
@@ -43,28 +43,41 @@ export default class PkiConfigurationEditComponent extends Component<Args> {
   @service declare readonly version: VersionService;
 
   @tracked invalidFormAlert = '';
-  @tracked errorBanner = '';
+  @tracked errorBanner: object[] = [];
 
   get isEnterprise() {
     return this.version.isEnterprise;
   }
 
+  async performSave() {
+    for (const modelName of ['cluster', 'acme', 'urls', 'crl']) {
+      const model = this.args[modelName as keyof Args];
+      // only attempt save if user has permission
+      if (!model.canSet) continue;
```

Comment:
I might not have enough context here but `canSet` makes me think the user has permission. Would you want to break the loop instead of continue if `canSet` is falsy?

Reply:
Yep - if `canSet = false` then the loop breaks via `continue` and move on to the next iteration of the for loop
<img width="1081" alt="Screenshot 2023-06-06 at 1 36 32 PM" src="https://github.com/hashicorp/vault/assets/68122737/d3bc9f6e-82a0-4449-8b87-43dccbc59ea2">

Reply:
Yep! Exactly. The other models should still attempt to save. Thanks for asking, I meant to include a screenshot to show the form with the empty state to clarify that a user can have mixed-permissions and therefore perform some edit operations